test/runtests.jl

```diff
@@ -1,14 +1,25 @@
-using Colors, FixedPointNumbers, ImageAxes, MappedArrays, Base.Test
+using Colors, FixedPointNumbers, MappedArrays, Base.Test, ImageCore, AxisArrays
+using Compat
+
+ambs = detect_ambiguities(ImageCore,AxisArrays,Base,Core)
+using ImageAxes
+ambs = setdiff(detect_ambiguities(ImageAxes,ImageCore,AxisArrays,Base,Core), ambs)
+if !isempty(ambs)
+    println("Ambiguities:")
+    for a in ambs
+        println(a)
+    end
+end
+@test isempty(ambs)
 
 if VERSION < v"0.6.0-dev"
```

Comment:
would it be better to be more conservative with this and set this to a known-working point, so it just doesn't test the inference on earlier versions as opposed to failing there?